packages/ui-components/src/components/ComboBoxOption/ComboBoxOption.component.tsx

```diff
@@ -60,7 +66,9 @@ export const ComboBoxOption = ({ children, disabled = false, value = "", label,
 
   // send option metadata to the ComboBox parent component via Context
   useEffect(() => {
-    addOptionValueAndLabel(value, label, children)
+    if (addOptionValueAndLabel) {
+      addOptionValueAndLabel(value, label, children)
+    }
   }, [value, label, children])
```

Comment:
**(Future Improvement Question, Not Blocking)**

Should `addOptionValueAndLabel` be in dependencies? 
Probably don't change now, in case. Just wanna highlight incase we can note in improvements.

Reply:
Done

Reply:
I had to revert this particular change as it was breaking tests - onclick handlers did not execute and test was hanging on waitFor button click.